common/app/model/meta.scala

```diff
@@ -414,12 +414,24 @@ case class TagCombiner(
   pagination: Option[Pagination] = None
 ) extends StandalonePage {
 
+  //We only use the left tag for CommercialProperties
+  val commercialPropertiesEditionAdTargeting: Seq[EditionAdTargeting] =
+    leftTag.properties.commercial.map(_.editionAdTargetings).getOrElse(Nil)
+  val commercialPropertiesEditionBrandings: Seq[EditionBranding] =
```

Comment:
Small point but these val names are very long!  Could possibly be defined inline in `metadata.make`?  But not particularly important.